deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -284,14 +284,31 @@ class AccountSetupController: UITableViewController, ProgressAlertHandler {
     }()
 
     lazy var mvboxMoveCell: SwitchCell = {
-        return SwitchCell(
+        let cell = SwitchCell(
             textLabel: String.localized("pref_auto_folder_moves"),
             on: dcContext.getConfigBool("mvbox_move"),
             action: { cell in
+                self.dcAccounts.stopIo()
                 self.dcContext.setConfigBool("mvbox_move", cell.isOn)
+                self.dcAccounts.startIo()
+        })
+        cell.uiSwitch.isEnabled = !dcContext.getConfigBool("only_fetch_mvbox")
```

Comment:
i'd leave the option enabled. the options do not really belong together, see recent comments in dev chat. android also does not disable things here.

also ui-wise it looks weird if that is enabled, does not make sense, and you cannot disable.